gravitee-apim-console-webui/src/components/alerts/alertEnginePromotion/_alert-engine-promotion.scss

```diff
@@ -0,0 +1,156 @@
+.ae-promo-content {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 150px 4px 4px !important;
+  gap: 4px;
+  transform: translate(0%, -20%);
+}
+
+.dashboard-promo-content {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 4px;
+  transform: translate(0%, 2%);
+}
+
+.ae-promo {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 32px;
+  width: 557px;
+  height: 518px;
+  order: 0;
+}
+
+.content {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: 24px;
+  width: 557px;
+  height: 450px;
+  order: 1;
+}
+
+.image-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 16px;
+  gap: 4px;
+  width: 557px;
+  height: 242px;
+  background: #606274;
+  border-radius: 4px 4px 0px 0px;
+  order: 0;
+}
+
+.info-content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 24px 24px 32px;
+  gap: 16px;
+  width: 557px;
+  height: 208px;
+  background: #ffffff;
+  border-radius: 0px 0px 4px 4px;
+  order: 1;
+}
+
+.button-content {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 20px;
+  width: 509px;
+  height: 36px;
+  order: 2;
+}
+
+.promotion-image {
+  width: 336px;
+  height: 210px;
+  order: 0;
+}
+
+.promo-title {
+  width: 265px;
+  height: 36px;
+  font-family: 'Golos UI', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 26px;
+  line-height: 36px;
+  color: #100c27;
+}
+
+.promo-label {
+  width: 108px;
+  height: 20px;
+  font-family: 'Golos UI', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.4px;
+  color: #100c27;
+}
+
+.text-paragraph {
+  width: 460px;
+  height: 27px;
+  font-family: 'Golos UI', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 16px;
+  color: #100c27;
+  display: flex;
+}
+
+.text-span {
+  width: 460px;
+  font-family: 'Golos UI', sans-serif;
```

Comment:
Is there a reason not to use `gravitee-ui-particles` for all fonts (https://particles.gravitee.io/?path=/story/theme-typography--showcase)

This will save us a lot of `css` 😁

Reply:
this css removed as part of our clean up process